tools/exports.js

```diff
@@ -17,3 +17,4 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["EXPECT_DIRECTIVE"] = EXPECT_DIRECTIVE;
```

Comment:
I think copying the regex with a var of the same name and a adding a comment that it must remain in sync with the other source file is a good workaround.